src/lib/utils/validatesInputs.tsx

```diff
@@ -45,6 +45,21 @@ export const validateAddressInput = (value: string, callback: React.Dispatch<Err
   callback(response);
 };
 
+const MIN_ALLOWED_VALIDATORS = 1;
+const MAX_ALLOWED_VALIDATORS = 100;
+
+export const validateValidatorsCount = (value: string, callback: React.Dispatch<ErrorObject>): void => {
+  const response = { shouldDisplay: true, errorMessage: '' };
+  let num: number = Number(value);
+  if (isNaN(num) || num < MIN_ALLOWED_VALIDATORS || num > MAX_ALLOWED_VALIDATORS){
+    response.errorMessage = 'Invalid validators count. Only number between 1-100 allowed';
```

Comment:
Use constant instead of hardcoded range in this sentence

Reply:
done